internal/uvm/create_lcow.go

```diff
@@ -166,6 +166,11 @@ func CreateLCOW(ctx context.Context, opts *OptionsLCOW) (_ *UtilityVM, err error
 		vpmemMaxCount:       opts.VPMemDeviceCount,
 		vpmemMaxSizeBytes:   opts.VPMemSizeBytes,
 	}
+
+	if !opts.AllowOvercommit && !opts.EnableDeferredCommit {
+		uvm.physicallyBacked = true
+	}
```

Comment:
Since we already have support for verifying options and really `AllowOvercommit` is the only thing necessary for a VM to be physically backed, I think this should be updated to be just 
```
uvm.physicallyBacked = !opts.AllowOvercommit
```

Reply:
good point